java/code/src/com/redhat/rhn/domain/kickstart/builder/KickstartBuilder.java

```diff
@@ -368,14 +366,6 @@ private void storeScript(String prefix, KickstartData ksData, StringBuilder buf,
 
     }
 
-    private void checkRoles() {
```

Comment:
Shouldn't we check in here that the user is part of these groups? Or this is not needed anymore?

Reply:
The same logic is enforced with RBAC filters now.